### ISSUE TITLE
how can I customize alert sounds

### ISSUE THREAD

**Issue by hellomyzn** [julienXX/terminal-notifier#299]
I wanna add sounds(aiff files) into '/System/Library/Sounds' to select my sounds when it notifies me.  but it is read-only.
so I wanna change the directory which this system reads
from: /System/Library/Sounds/
to     : ~/Library/Audio/Sounds/Alerts/

Please let me know if someone has any clue.

**Comment by RococoCoding** [comment]
I was also looking into this. I haven't found a way to do it through terminal-notifier, but I was able to have my script run 'afplay honk.aiff' after the notification, which works pretty well!

**Comment by julienXX** [comment]
Yes sorry but it does not seem possible with the current APIs. Closing the issue for now.
